.github/workflows/corona-charts.yml

```
@@ -37,6 +37,7 @@ jobs:
       - name: Run R scripts
         run: |
           Rscript owid_vaccinations_worldwide/owid_vaccinations_worldwide.R
+          Rscript corona-auto-ch/corona-auto-ch-test.R
```

@nthelitz  Ich würde das R script gleich wie der Ordner nennen. Das ist ja jetzt kein Test mehr oder hattest du da noch einen anderen Gedanken dabei?

```suggestion
          Rscript corona-auto-ch/corona-auto-ch.R
```

---

☑️